client/src/browse/list/ItemCard.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import { BsCardImage } from "react-icons/bs";
+
+const ItemCard = (props) => {
+  const clickHandler = () => {
+    props.setSelectedItem(props.item.id);
+  };
+
+  return (
+    <Container
+      id={props.item.id}
+      onClick={clickHandler}
+      style={{
+        backgroundColor: props.selectedItem === props.item.id ? "red" : "blue",
+      }}
```

Comment:
@MarkAlperin I see that you're using inline styling to conditionally render, Im not sure if you're aware that styled comp's have this functionality built in.

`

//Im going to assume props has been destructered 
//so props.selectedItem = selectedItem && item = props.item

<Container
  id={item.id}
  onClick={clickHandler}
  isSelected={selectedItem === item.id}
>

const Container = styled.div`
  min-width: 150px;
  max-width: 150px;
  height: 150px;
  margin: 10px;
  cursor: pointer;
  background-color: ${({ isSelected } => isSelected ? "red" : "blue" )};
`;  

`
This might be an approach I would take, but Im sure both work. I would like to hear your thoughts. Whether it is on here or later today.